csrc/index_compute.cpp

```diff
@@ -3460,12 +3464,14 @@ Val* Index::eye(
 //           \               \-> I4
```

Comment:
Maybe we could improve readability of the code:
```
// -> I0 -> split -> I1 -> split -> I3 (Bulk)
//              \               \-> I4
//               \-> I2 -> split -> I5 (Bulk)
//                              \-> I6
// -> I7
```

At the beginning I though that `I7` is some kind of artifact.

Reply:
Added in https://github.com/NVIDIA/Fuser/pull/2004 in https://github.com/NVIDIA/Fuser/pull/2004/commits/f7869f4de4527b0a449470d84bf31d65fa0e2763. I am resolving reviews in followup PRs so that I do not need to handle merge conflicts.